### DISCUSSION TITLE
nonebot使用反向ws连接onebot-mirai，回复消息报错

### DISCUSSION THREAD

**Discussion by WesternLi** [nonebot/discussions#443] [问答]
nonebot使用反向ws连接onebot-mirai可以连接上，并且nonebot可以看到qq消息内容。
但只要涉及到消息回复就会报错，机器人无法回复消息
以下截图中向机器人发送的指令为 /echo 1
nonebot报错截图
![图片](https://github.com/nonebot/discussions/assets/61825420/2be710b3-41fc-49d2-8d7c-72a9661fbce8)

mirai报错截图
![图片](https://github.com/nonebot/discussions/assets/61825420/50ab5247-88ba-42d5-ba19-c512f0207191)

onebot-mirai配置文件
![图片](https://github.com/nonebot/discussions/assets/61825420/c0ec5025-3efb-4254-a702-2b2322f066e1)

**Comment by yanyongyu** [comment]
这个报错看上去是你协议端的问题，与nonebot无关
